cli/config/configuration-qemu.toml.in

```diff
@@ -108,6 +108,14 @@ virtio_fs_daemon = "@DEFVIRTIOFSDAEMON@"
 # Default size of DAX cache in MiB
 virtio_fs_cache_size = @DEFVIRTIOFSCACHESIZE@
 
+# Extra args for virtiofsd daemon
+#
+# Format example:
+#   "-o arg1=xxx,arg2 --arg3=yyy"
+#
+# see `virtiofsd -h` for possible options.
+# virtio_fs_extra_args = ""
```

Comment:
Does this option need to be added to any of the other `cli/config/configuration*.toml.in` files?

Reply:
Thanks for the reminder, let me play the other hypervisor a bit and see if they can support.

Reply:
We only support fc with qemu/nemu. There is no need to update other hypervisor's config.toml, which would only confuse users.

Reply:
virtio-fs upstream seems only support qemu now, so it's not necessary to update other hypervisor config template so far.

Reply:
@renzhengeek while it is true that upstream virtiofs will only work with QEMU, today's kata enables virtiofs using nemu. Could we add this option to nemu config as well? 

@bergwolf @stefanha WDYT?